### PR TITLE
implemented stopwatch

### PR DESCRIPTION
**Pull request by ihorandrianov**
- [DEMO LINK](https://ihorandrianov.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/blocks/_watch.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  margin-top: 50px;
```
Comment (grafit-off) [inline src/styles/blocks/_watch.scss:6]: It's a BEM block, you can't set margins on it

File: src/styles/blocks/_watch.scss [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  &__center-point {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    @include circle(20px);
+
+    background: $primary-color;
+
+    animation: blobs 15s ease-in-out infinite alternate;
```
Comment (grafit-off) [inline src/styles/blocks/_watch.scss:85]: You can use placeholders or mixins for repeated properties like animation here.

File: src/styles/blocks/_watch.scss [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  margin-top: 50px;
+
+  &__circle {
+    @include circle(600px);
+
+    background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+    background-size: 400% 400%;
+
+    animation: blobs 15s ease-in-out infinite alternate,
+      bg-gradient 15s ease infinite;
```
Comment (grafit-off) [inline src/styles/blocks/_watch.scss:15]: You can use variables for animation duration, for the reason that this value is the same

File: src/styles/blocks/_watch.scss [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    transition: 2s;
```
Comment (grafit-off) [inline src/styles/blocks/_watch.scss:16]: You should set transition properties that are going to change, so the browser doesn't need to watch all properties

File: src/styles/utils/_keyframes.scss [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  50% {
+    transform: rotate(180deg);
+  }
+  75% {
+    transform: rotate(270deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```
Comment (grafit-off) [inline src/styles/utils/_keyframes.scss:53]: In my opinion, you can make a mixin for this case

**Review by grafit-off — APPROVED** [review]
Good job 🔥 , awesome animation 👍

File: src/styles/blocks/_watch.scss (hunk 1 above)
Comment (Denchepornyuk) [reply src/styles/blocks/_watch.scss:6]: yep, you can't add margin to BEM block, remove it

File: src/styles/blocks/_watch.scss (hunk 3 above)
Comment (Denchepornyuk) [reply src/styles/blocks/_watch.scss:15]: agree, create variable sounds good

File: src/styles/blocks/_watch.scss (hunk 4 above)
Comment (Denchepornyuk) [reply src/styles/blocks/_watch.scss:16]: agree, let's fix it

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
Awesome! ⭐️

**Pull request closed without merging**